### DISCUSSION TITLE
v2.0.0 Towards the Future

### DISCUSSION THREAD

**Discussion by ThePharaohArt** [ArtexJay/Obsidian-CyberGlow#94] [Announcements]
**v2.0.0 Changelog**

23 New Addition, 14 Fixes, 4 Removal, 25 Changes

  **Additions**
 - Added: animation when hovering over toggle
 - Added: Captions
 - Added: Custom Bold Text Color/Glow Style Settings
 - Added: Custom color on hover
 - Added: Delete button to be more consistent with the delete style
 - Added: Graph Control Scollbar (Disabled in Vanilla)
 - Added: Grid
 - Added: Highlightr Plugin Styling
 - Added: Infobox Callout
 - Added: Page highlight option in style settings
 - Added: Return of Indent lines in code block now that Obsidian fixed the graphical bug
 - Added: Style Kith compatible with ITS
 - Added: Style Settings Rainbow Folder option
 - Added: Style Settings toggle to enable styling only on active tab, for easier identification of active tab
 - Added: Style for Mermaid Tool
 - Added: Styled ctrl hover file popups
 - Added: Styled Progress bar when loading Obsidian
 - Added: Themed Elements that that become focused
 - Added: Themed flashing for finding text
 - Added: Themed text search highlight
 - Added: Theming for Editing Toolbar Plugin
 - Added: Toggle to disable default folder icons Style Settings
 - Added: Toggle to show vanilla chevron Style Settings
 
  **Fixed**
 - Fixed: Bullet list bugs including in source view
 - Fixed: Button style light mode
 - Fixed: Chevron improperly hidden
 - Fixed: Custom Font style settings
 - Fixed: Default Statusbar view
 - Fixed: Dropdown Menu text from being unreadable in certain circumstances
 - Fixed: Graphic Bug for dragging ghost
 - Fixed: Inlink color
 - Fixed: Line Highlight bug when gutter number line is disabled
 - Fixed: Minor styling with stacked tabs
 - Fixed: Note background color and transparency is more consistent with that of the sidepanels
 - Fixed: Sidebar arrow buttons overlap
 - Fixed: Some minor graphical bugs in the settings page
 - Fixed: Theme lag by identified and removed code that increased PC resource unnecessarily usage (It was a looping animation)
 
  **Removed**
 - Removed: Colored Text in order to improve readability
 - Removed: Due to Sidepanel animations on mobile not triggering properly
 - Removed: Inactive note text color
 - Removed: Inactive tab text color
 
 **Updated**
 - Updated: Background Image to now extend into Titlebar when "Obsidian Frame" is selected
 - Updated: Blockquotes style to be less bulky
 - Updated: Canvas style
 - Updated: Colors and style for Quote blockquote
 - Updated: Custom active note title color
 - Updated: Custom blockquote color
 - Updated: Custom highlight color
 - Updated: Custom tab color
 - Updated: Disabled Toggle switch
 - Updated: Hover and active Pages in File Explorer
 - Updated: Italic, Bold, Strikeout texts
 - Updated: Kanban Styling
 - Updated: Light mode colors
 - Updated: Mermaid Diagrams
 - Updated: Overhauled the setting style
 - Updated: Phone Notes
 - Updated: Phone Settings
 - Updated: Phone Sidepanels
 - Updated: Phone Tab Scroll Style
 - Updated: Sidebar arrow buttons overlap
 - Updated: Table with clear differentiator for table title cells
 - Updated: Tablet nav header
 - Updated: Tablet Pane Spacing bringing it in line with Desktop version
 - Updated: Tablet Sidepanels with better pinned/non pinned sidepanel design
 - Updated: Tag style
 - Updated: Textbox style

<hr /><em>This discussion was created from the release <a href='https://github.com/ArtexJay/Obsidian-CyberGlow/releases/tag/v2.0.0'>v2.0.0 Towards the Future</a>.</em>

**Comment by Voidraizer** [comment]
Hi, I really like your theme but I have a couple questions. In this update

> Added: Style for Mermaid Tool

What does this mean? How do we style Mermaid? I as because [this is  pretty hard to make out](https://i.imgur.com/MsQIt7A.png). I don't really want to use a background but I did just discover that by enabling it and setting these options, it becomes [readable](https://i.imgur.com/M2VKScj.png) but I wouldn't think this is intended behavior.

I'm also really curious to know the source behind the colored blocks in the [Cyber Glow Preview](https://i.imgur.com/vC4OISa.png) and finally the quote block [here](https://i.imgur.com/H9hLJG3.png). I [dabbled a little](https://i.imgur.com/FbNIlrn.png) but it doesn't look remotely similar.

Man there are so many little tricks I'm excited to learn because I really like this theme and obsidian in general! Thank you for creating this awesome theme!

**Reply by ThePharaohArt** [reply]
It was supposed to be styled in line with the theme, but I removed it due to it causing issues and that readability issue that you mentioned.
Obsidian supports 2 kinds of quote boxes, the one you dabbled in was a callout quote, the one in the old Cyber Glow Preview was the traditional quote box which you can get to with by adding ">" symbol to the beginning of the text. However, after v2.0.0 it was changed to fit more the design of the theme, here are both quote boxes currently in the theme
<img width="679" height="405" alt="image" src="https://github.com/user-attachments/assets/3991349f-2fe3-4c92-a29d-30f99d73e34f" />

Only now realized that simple (ie non-collapsible) callout looks bit broken which I will fix.
<img width="681" height="223" alt="image" src="https://github.com/user-attachments/assets/93f1fd2c-00c5-4ca5-bbc5-3e8a8a82fb70" />



``` 
> [!quote] Title
> Test
> <cite>Test</cite>
```

Adding a plus or minus sign makes it collapsible.
```
> [!quote]+ Title
> Test
> <cite>Test</cite>
```
